### PR TITLE
Fix formatting (scalafmt & scalafix)

### PR DESCRIPTION
**Pull request by tpetillot**

### CODE REVIEW

File: examples/src/main/scala/io/finch/iteratee/Main.scala [hunk 1]
```diff
@@ -11,15 +9,14 @@ import io.finch.catsEffect._
...
+
+/** A Finch application featuring iteratee-based streaming support. This approach is more advanced and performant then basic
+  * [[com.twitter.concurrent.AsyncStream]]
   *
   * There are three endpoints in this example:
   *
-  *  1. `sumJson` - streaming request
-  *  2. `streamJson` - streaming response
-  *  3. `isPrime` - end-to-end (request - response) streaming
+  *   1. `sumJson` - streaming request 2. `streamJson` - streaming response 3. `isPrime` - end-to-end (request - response) streaming
```
Comment (joroKr21) [inline examples/src/main/scala/io/finch/iteratee/Main.scala:19]: Broken formatting

File: examples/src/main/scala/io/finch/wrk/Finagle.scala [hunk 2]
```diff
@@ -8,17 +8,13 @@ import com.twitter.io.Buf
 import com.twitter.util.Future
 import io.finch.internal._
 
-/**
-  * How to benchmark this:
+/** How to benchmark this:
   *
-  * 1. Run the server: sbt 'examples/runMain io.finch.wrk.Finagle'
-  * 2. Run wrk: wrk -t4 -c24 -d30s http://localhost:8081/
+  *   1. Run the server: sbt 'examples/runMain io.finch.wrk.Finagle' 2. Run wrk: wrk -t4 -c24 -d30s http://localhost:8081/
```
Comment (joroKr21) [inline examples/src/main/scala/io/finch/wrk/Finagle.scala:13]: Broken formatting

File: examples/src/main/scala/io/finch/wrk/Finagle.scala [hunk 3]
```diff
@@ -8,17 +8,13 @@ import com.twitter.io.Buf
...
   *
-  * Rule of thumb for picking values for params `t` and `c` (given that `n` is a number of logical
-  * cores your machine has, including HT):
+  * Rule of thumb for picking values for params `t` and `c` (given that `n` is a number of logical cores your machine has, including HT):
   *
-  *   t = n
-  *   c = t * n * 1.5
+  * t = n c = t * n * 1.5
```
Comment (joroKr21) [inline examples/src/main/scala/io/finch/wrk/Finagle.scala:17]: Broken formatting

File: examples/src/main/scala/io/finch/wrk/Finch.scala [hunk 4]
```diff
@@ -5,17 +5,13 @@ import io.circe.generic.auto._
 import io.finch._
 import io.finch.circe._
 
-/**
-  * How to benchmark this:
+/** How to benchmark this:
   *
-  * 1. Run the server: sbt 'examples/runMain io.finch.wrk.Finch'
-  * 2. Run wrk: wrk -t4 -c24 -d30s http://localhost:8081/
+  *   1. Run the server: sbt 'examples/runMain io.finch.wrk.Finch' 2. Run wrk: wrk -t4 -c24 -d30s http://localhost:8081/
```
Comment (joroKr21) [inline examples/src/main/scala/io/finch/wrk/Finch.scala:10]: Broken formatting

File: examples/src/main/scala/io/finch/wrk/Finch.scala [hunk 5]
```diff
@@ -5,17 +5,13 @@ import io.circe.generic.auto._
...
   *
-  * Rule of thumb for picking values for params `t` and `c` (given that `n` is a number of logical
-  * cores your machine has, including HT):
+  * Rule of thumb for picking values for params `t` and `c` (given that `n` is a number of logical cores your machine has, including HT):
   *
-  *   t = n
-  *   c = t * n * 1.5
+  * t = n c = t * n * 1.5
```
Comment (joroKr21) [inline examples/src/main/scala/io/finch/wrk/Finch.scala:14]: Broken formatting

**Review by joroKr21 — COMMENTED** [review]
I just point out the places where formatting broke the doc and need to be fixed by hand

**Comment by tpetillot** [comment]
@joroKr21 will happend everytime with format on compile https://github.com/finagle/finch/pull/1466, shouldn't we find a way to prevent this formatting?

**Comment by joroKr21** [comment]
Is it fine if I push a fix to your branch?

**Review by joroKr21 — APPROVED** [review]

**Pull request merged**